mne/simulation/source.py

```diff
@@ -328,9 +328,9 @@ def simulate_stc(
             d = len(v) - len(np.unique(v))
             if d > 0:
                 raise RuntimeError(
-                    "Labels had %s overlaps in the %s "
+                    f"Labels had {d} overlaps in the {hemi} "
                     "hemisphere, "
-                    "they must be non-overlapping" % (d, hemi)
+                    "they must be non-overlapping"
```

Comment:
Please combine with previous line.

Reply:
I updated the suggested changes.
Thanks